.github/workflows/parsegithubissue.yaml

```diff
@@ -42,29 +36,23 @@ jobs:
           issue-number: ${{ github.event.issue.number }}
           body: |
             Requester cannot also be approver - please seek a second approver.
-
     #Remove any approval labels if approver is equal to requester
       - name: Remove approved label
         if: contains(github.event.issue.labels.*.name, 'approved') && github.event.issue.user.login == github.actor || contains(github.event.issue.labels.*.name, 'auto-approved')
-        uses: actions-ecosystem/action-remove-labels@v1
-        with:
-          labels: |
-            approved
-            auto-approved
-
+        run: gh issue edit "$NUMBER" --remove-label "$LABELS"
+        env:
```

Comment:
Does this work without a for loop? When I was adding multiple labels in a test branch I needed a for loop

Reply:
Will check with Louise if that was tested - good spot

Reply:
Tested multiple labels and this works without the need for a logical loop.